packages/oui-checkbox/checkbox.less

```diff
@@ -4,6 +4,7 @@
 .oui-checkbox {
   @oui-checkbox-selector: oui-checkbox;
   @oui-icon-selector: oui-icon-checkbox;
+  position: relative;
```

Comment:
In order to make consistency, please refer move this rule directly in the `.checkbox` mixin.

ref: https://github.com/ovh-ux/ovh-ui-kit/blob/v2.x/packages/oui-checkbox/_mixins.less#L84